### PR TITLE
Added tests for AdminService

### PR DESCRIPTION
**Pull request by EnKarin**

### CODE REVIEW

File: src/main/java/io/github/enkarin/bookcrossing/registation/dto/UserDto.java [hunk 1]
```diff
@@ -1,40 +1,55 @@
...
     @Schema(description = "Почта", example = "al@yandex.ru")
     @Email(message = "email: Некорректный почтовый адрес")
-    private String email;
+    private final String email;
 
     @Schema(description = "Город", example = "Новосибирск")
-    private String city;
+    private final String city;
+
+    private UserDto(final String name, final String login, final String password, final String passwordConfirm,
```
Comment (mfvanek) [inline src/main/java/io/github/enkarin/bookcrossing/registation/dto/UserDto.java:40]: Can be generated via lombok:
`@AllArgsConstructor(access = AccessLevel.PRIVATE)`
See also https://projectlombok.org/features/constructor

File: src/test/java/io/github/enkarin/bookcrossing/admin/service/AdminServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        usersId.add(user2.getUserId());
+        assertThat(adminService.findAllUsers(0))
+                .hasSize(2)
+                .containsExactlyInAnyOrder(InfoUsersDto.fromUser(user1, 0),
+                        InfoUsersDto.fromUser(user2, 0));
+    }
+
+    @Test
+    void findAllUsersEmpty() {
+        assertThat(adminService.findAllUsers(0)).hasSize(0);
```
Comment (mfvanek) [inline src/test/java/io/github/enkarin/bookcrossing/admin/service/AdminServiceTest.java:74]: Replace` hasSize(0)` with` isEmpty()`

File: src/test/java/io/github/enkarin/bookcrossing/books/service/BookServiceTest.java [hunk 3]
```diff
@@ -43,39 +46,61 @@ void saveExceptionTest() {
                 .hasMessage("Пользователь не найден");
     }
 
-    @SqlGroup({
-        @Sql("classpath:db/scripts/insert_user.sql"),
-        @Sql("classpath:db/scripts/insert_books.sql")
-    })
     @Test
     void findBookForOwnerTest() {
+        final User user1 = userService.saveUser(UserDto.create("Tester", "user", "123456",
```
Comment (mfvanek) [inline src/test/java/io/github/enkarin/bookcrossing/books/service/BookServiceTest.java:51]: Consider to use https://projectlombok.org/features/experimental/SuperBuilder on DTO's and in tests

**Review by mfvanek — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/enkarin/bookcrossing/registation/dto/UserDto.java (hunk 1 above)
Comment (EnKarin) [reply src/main/java/io/github/enkarin/bookcrossing/registation/dto/UserDto.java]: Changed

File: src/test/java/io/github/enkarin/bookcrossing/admin/service/AdminServiceTest.java (hunk 2 above)
Comment (EnKarin) [reply src/test/java/io/github/enkarin/bookcrossing/admin/service/AdminServiceTest.java]: Changed

File: src/test/java/io/github/enkarin/bookcrossing/books/service/BookServiceTest.java (hunk 3 above)
Comment (EnKarin) [reply src/test/java/io/github/enkarin/bookcrossing/books/service/BookServiceTest.java]: Added

**Pull request merged**